event_routing_backends/helpers.py

```diff
@@ -175,3 +175,17 @@ def backend_cache_ttl():
         int
     """
     return getattr(settings, 'EVENT_TRACKING_BACKENDS_CACHE_TTL', 600)
+
+
+def get_business_critical_events():
+    """
+    Return list of business critical events.
+
+    Returns:
+        list
+    """
+    return getattr(settings, 'BUSINESS_CRITICAL_EVENTS', [
```

Comment:
@rehanedly can we prefix setting name with `EVENT_TRACKING_BACKENDS_`?

Reply:
@ziafazal done